what4/what4.cabal

```diff
@@ -236,6 +236,17 @@ executable quickstart
     parameterized-utils,
     what4
 
+executable abduct
+  main-is: test/Abduct.hs
+  default-language: Haskell2010
+
+  build-depends:
+    base,
+    parameterized-utils,
+    what4,
+    bimap >= 0.2,
```

Comment:
I don't think this dependency is used anywhere in the `abduct` executable, unless I'm missing something.